tests/integration/README.md

```diff
@@ -0,0 +1,134 @@
+# Cloud Provider Integration Tests
+
+## Requirements
+
+### Amazon Web Services
+
+- AWS account with access key ID and secret access key with permissions to
+    create EKS cluster and ECR repository.
+- AWS CLI, need not be configured with the AWS account.
```

Comment:
Do you mean:
a) It **does not need** to be configured with the AWS account
or
b) It **must not** be configured with the AWS account?

Reply:
I would then rephrase it to something like:

```suggestion
- AWS CLI, which does not need to be configured with the AWS account above.
```